src/abr/puffer_ttp.cc

```diff
@@ -7,19 +7,20 @@ PufferTTP::PufferTTP(const WebSocketClient & client,
                      const string & abr_name, const YAML::Node & abr_config)
   : Puffer(client, abr_name, abr_config)
 {
+  std::cerr << "abr_name = " << abr_name_ << endl;
   /* load neural networks */
   if (abr_config["model_dir"]) {
     fs::path model_dir = abr_config["model_dir"].as<string>();
-
+    cerr << "model_dir=" << abr_config["model_dir"] << endl;
```

Comment:
Nit (and fix if you want): You can perform stream output directly on `model_dir`, the `fs::path` variable.
```
cerr << "model_dir=" << model_dir << endl;
```